message_sender/tests.py

```diff
@@ -452,6 +455,139 @@ def test_event_nack_last(self):
         #     self.check_logs("Metric: 'vumimessage.maxretries' [sum] -> 1"))
 
 
+@override_settings(MESSAGE_BACKEND='junebug',
+                   ROOT_URLCONF='message_sender.urls')
+class TestJunebugMessagesAPI(AuthenticatedAPITestCase):
+    def setUp(self, *args, **kwargs):
+        reload(sys.modules[settings.ROOT_URLCONF])
```

Comment:
In our urls file, we are setting the route for `/api/v1/events` according to `EventListenerFactory.create()`, so if we just override the setting, it still points to the old vumi event view, not the new Junebug one, so we have to reload the URLs for it to pick up the changed setting and point to the right view.